openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx

```diff
@@ -98,6 +98,7 @@ const GlossaryHeader = ({
     version: string;
   }>();
   const { fqn } = useFqn();
+  const { id } = useParams<{ id: string }>();
```

Comment:
@Ashish8689 , make sure to include this in effect dependency from where we are calling the `fetchCurrentGlossaryInfo` function.